contracts/neutron_interchain_txs/src/lib.rs

```diff
@@ -17,5 +17,4 @@
 pub mod contract;
```

Comment:
in some lib/mod files neutron license is presented, in some it isn't. I suppose it should be in all such files?